src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +8,46 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val genericGreetings = listOf("Hello world", "Salut tout le monde", "Hola, mundo")
+
+        val EnglishGreeting: HelloWorldGreeting = object : HelloWorldGreeting {
```

Comment:
The variable name `EnglishGreeting` should start with a lowercase letter according to Kotlin naming conventions. Consider renaming it to `englishGreeting`.